pkg/controller/container-runtime-config/helpers.go

```diff
@@ -313,9 +317,9 @@ func createCRIODropinFiles(cfg *mcfgv1.ContainerRuntimeConfig) []generatedConfig
 			glog.V(2).Infoln(cfg, err, "error updating user changes for log-level to crio.conf.d: %v", err)
 		}
 	}
-	if ctrcfg.PidsLimit != 0 {
+	if *ctrcfg.PidsLimit >= 0 {
```

Comment:
Looking at the caller above, it seems possible that the pointer is `nil` at this point.

Reply:
fixed